README.md

```diff
@@ -105,7 +105,7 @@ forecast_rows: 5
 locale: en-GB
 time_pattern: HH:mm
 time_format: 24
-date_pattern: P
+date_pattern: ccc, d.MM.yy
```

Comment:
good catch, but would you please change the value to `D`, as this is the default date used if no other pattern is specified.